### DISCUSSION TITLE
Start docker server on boot

### DISCUSSION THREAD

**Discussion by IZeeLikeEyeZee** [ptrumpis/snap-camera-server#43] [Q&A] [answered]
Hey is there any way to set the docker container to start on boot? Since it seems that if snap cam is started on boot and docker isnt that I need to start docker, then restart snap cam for everything to work.

**Comment by ptrumpis** [comment, marked as answer]
[Replace line 25-26 in docker-compose.yml with this code](https://github.com/ptrumpis/snap-camera-server/blob/main/docker-compose.yml#L25)
```yaml
  webproxy:
    restart: always
    image: nginx
```

That should do the trick.

**Reply by Wonpil-Myoui** [reply]
Hi, sorry to reopen an old discussion, just wanted to ask why this doesn't work for mine? I still have to open Docker desktop and manually start the containers when I open my computer. I also find that Snap camera takes a little long to start whenever I quit then reopen the shortcut, take maybe 2-3 minutes everytime.

**Reply by ptrumpis** [reply]
Did you check

**Docker Desktop -> Settings -> Open Docker Dashboard at startup**

**Reply by Wonpil-Myoui** [reply]
yes but I still have to manually start it

**Reply by ptrumpis** [reply]
Ok, try this:

Open a new Terminal inside your server directory and run the command below to update the `docker-compose.yml` configuration:
```bash
docker compose up
```
